py/torch_tensorrt/__init__.py

```diff
@@ -90,13 +90,21 @@ def _find_lib(name: str, paths: List[str]) -> str:
 
 def _register_with_torch() -> None:
     trtorch_dir = os.path.dirname(__file__)
-    if os.path.isfile(trtorch_dir + "/lib/libtorchtrt.so"):
+    linked_file = (
+        "/torchtrt.dll" if sys.platform.startswith("win") else "/lib/libtorchtrt.so"
```

Comment:
This isnt enabled yet but I think the location for the dll will still be lib/torchtrt.dll